src/App.css

```diff
@@ -1,4 +1,7 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600&family=Roboto+Mono:wght@400;600&display=swap');
 *{
   padding: 0;
   margin: 0;
-}
\ No newline at end of file
+  scroll-behavior: smooth;
+  font-family: 'Poppins', 'Roboto Mono', sans-serif ,'monospace';
```

Comment:
Will all text throughout the site share this same font styling or will you be using more than one font/font style?